libs/common-services/src/main/java/uk/gov/di/ipv/core/library/service/ClientOAuthSessionDetailsService.java

```diff
@@ -51,8 +51,7 @@ public ClientOAuthSessionItem generateClientSessionDetails(
         clientOAuthSessionItem.setGovukSigninJourneyId(
                 claimsSet.getStringClaim("govuk_signin_journey_id"));
         clientOAuthSessionItem.setVtr(claimsSet.getStringListClaim("vtr"));
-        Boolean reproveIdentity =
-                Boolean.parseBoolean(claimsSet.getStringClaim("reprove_identity"));
+        Boolean reproveIdentity = claimsSet.getBooleanClaim("reprove_identity");
```

Comment:
What is the behaviour of this if the value is null/missing? do we have some unit test cases